### ISSUE TITLE
dhewm doesn't detect my savegames from Doom 3 installation

### ISSUE THREAD

**Issue by SingingBush** [dhewm/dhewm3#216]
I've had Doom 3 installed on Linux before using the proper installer that put the game into `/usr/local/games/doom3` and would put _DoomConfig.cfg_, additional maps, the license key, and game saves under `~/.doom3/base/`.

However, when I run dhewm3 these saves are not available.

**Comment by DanielGibson** [comment]
yeah, dhewm3 saves elsewhere and I doubt the savegames are even compatible.
you could try though by copying them to `~/.local/share/dhewm3/base/savegames/`

the config belongs in `~/.config/dhewm3/base/`

**Comment by SingingBush** [comment]
awesome, I copied `cp -r ~/.doom3/base/savegames/ ~/.local/share/dhewm3/base/` booted dhewm3 up and was able to continue from the old save files without any issue. Maybe worth allowing the UI to list games that are in the old directory location as a convenience or at least document this step somewhere.
